src/Compiler/Microsoft.AspNetCore.Razor.Language/src/DefaultBoundAttributeDescriptor.cs

```diff
@@ -13,7 +13,7 @@ internal sealed class DefaultBoundAttributeDescriptor : BoundAttributeDescriptor
         bool hasIndexer,
         string? indexerNamePrefix,
         string? indexerTypeName,
-        string? documentation,
+        object? documentationObject,
```

Comment:
Could we assert what the type of this is?

Reply:
Sure thing.